priv/templates/install/seeds.exs

```diff
@@ -100,3 +100,155 @@ Content.publish_layout(layout)
+  component_category_id: footers.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Footer 2",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+  component_category_id: footers.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Footer 3",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+  component_category_id: footers.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Footer 1",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+  component_category_id: stats.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Footer 2",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+  component_category_id: stats.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Footer 3",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/footers/01_1648bd354f.png",
+  component_category_id: stats.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Sign In 1",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/sign-in/01_b25eff87e3.png",
+  component_category_id: sign_ins.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Sign In 2",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/sign-in/01_b25eff87e3.png",
+  component_category_id: sign_ins.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Sign In 3",
+  thumbnail: "https://static.shuffle.dev/components/preview/43b384c1-17c4-470b-8332-d9dbb5ee99d7/sign-in/01_b25eff87e3.png",
+  component_category_id: sign_ins.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Title",
+  thumbnail: "/component_thumbnails/title.jpg",
+  component_category_id: basics.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Button",
+  thumbnail: "/component_thumbnails/button.jpg",
+  component_category_id: basics.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Link",
+  thumbnail: "/component_thumbnails/link.jpg",
+  component_category_id: basics.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Paragraph",
+  thumbnail: "/component_thumbnails/paragraph.jpg",
+  component_category_id: basics.id
+})
+Beacon.ComponentDefinitions.create_component_definition!(%{
+  name: "Aside",
+  thumbnail: "/component_thumbnails/aside.jpg",
+  component_category_id: basics.id
+})
```

Comment:
I don't think think we need this block of code anymore, right?

```suggestion
```

Reply:
Removed